<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetProjectiles</class>
 <widget class="QWidget" name="WidgetProjectiles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>262</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Gravity accel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QDoubleSpinBox" name="gravity">
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>9.810000000000000</double>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Initial height</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDoubleSpinBox" name="height">
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="value">
      <double>30.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Angle</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QDoubleSpinBox" name="angle">
     <property name="minimum">
      <double>-90.000000000000000</double>
     </property>
     <property name="maximum">
      <double>90.000000000000000</double>
     </property>
     <property name="value">
      <double>45.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Initial speed</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QDoubleSpinBox" name="speed">
     <property name="maximum">
      <double>50.000000000000000</double>
     </property>
     <property name="value">
      <double>40.000000000000000</double>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="2">
    <widget class="QCheckBox" name="sameZ">
     <property name="text">
      <string>Same Z coordinate</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QCheckBox" name="trajectory">
     <property name="text">
      <string>Show trajectory</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;color:darkred</string>
     </property>
     <property name="text">
      <string>Solver 1</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="styleSheet">
      <string notr="true">font-weight:bold;color:darkblue</string>
     </property>
     <property name="text">
      <string>Solver 2</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="solver1">
     <item>
      <property name="text">
       <string>Euler</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Symplectic Euler</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Verlet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Midpoint</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RK-2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Velocity Verlet</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>RK-4</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="solver2"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
